raster-foundry/raster-foundry
# Merge page sorting parameters

## Overview

This PR adjusts the handling of sort parameters within our paging methods. Rather than directly merging in a doobie fragment that contains an `ORDER BY` clause, we now format the sort parameters as a `Map[String, Order]` and merge it into the sort parameters provided by the `PageRequest`. This prevents occurrence of multiple `ORDER BY`s.

### Checklist


### Notes

In order to avoid default args I introduced a new signature for the `page` method which should allow us to not need to provide empty maps everywhere extra sorting isn't necessary.

## Testing Instructions

 * list users in a platform, an organization, and a team and ensure there are no errors
 * list scenes in a project and ensure there are no errors

Closes #4115
